### DISCUSSION TITLE
read-only modifier always true for input type="range"

### DISCUSSION THREAD

**Discussion by hertg** [tailwindlabs/tailwindcss#13850] [Help]
It looks like the read-only modifier doesn't work properly on `range` inputs; it is always true.

The following code will lead to the input slider always having the `not-allowed` cursor, even when the input field is **not** marked readonly.

```html
<input type="range" class="read-only:cursor-not-allowed" />
```

Is this an issue in Tailwind, or an issue in the browser? I can reproduce it in current Firefox and Vivaldi. It is reproducible on https://play.tailwindcss.com/ with Tailwind version `v3.4.4` and `v4.0.0-alpha.16`.

**Comment by wongjn** [comment]
[It seems like `:read-only` is only really meant for text-editable field elements](https://stackoverflow.com/a/70375659/19655631).
